src/generic/colrdlgg.cpp

```diff
@@ -131,6 +144,17 @@ wxGenericColourDialog::wxGenericColourDialog(wxWindow *parent,
 
 wxGenericColourDialog::~wxGenericColourDialog()
 {
+#if USE_PREVIEW_WITH_ALPHA
```

Comment:
Why do we need to call `Unbind()` explicitly when the dialog is being destroyed anyhow? AFAICS this code shouldn't be necessary at all...